circle.yml

```diff
@@ -18,7 +18,7 @@ test:
     - npm run test-cov
 deployment:
   npm:
-    branch: master
+    tag: /[0-9]+.*/
```

Comment:
@yongpingchen 

I agree with kumano-san that using non number at the end of tag name is acceptable at to this [comment](https://github.com/KiiCorp/AndroidStorageSDK/pull/1946#discussion_r89045343). I expect `0beta2` can be used as a tag name.

According to the comment, I adopt the regular expression.

Could you talk about this with kumano-san? I follow the decision.